Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -400,6 +400,10 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _redirectToAttribute: function (attr) {
             var attrRedirectionUrl = this.get(attr);
 
+            this.set('publishRedirectionUrl');
```

Comment:
@sunpietro there are still 2 problems here:

1. as said before the way the attributes are reset (by setting `undefined`) is wrong
1. the place were you do it is wrong as well. As the name `_redirectToAttribute` suggests (and as written in its doc description), this method is supposed to redirect the user to the URI contained in the `attr` attribute passed in parameter. Resetting the 3 attributes while doing that is clearly not in the scope of that method.